Add PAYPAL_WEB_PROFILE_CONFIG to ecommerce defaults.

jimabramson opened the pull request:
XCOM-319

@rlucioni @clintonb FYI @Nickersoft related to https://github.com/edx/ecommerce/pull/177

I'm still looking for ops reviewers, but you can get started catching stuff that's missing here or just plain wrong.

clintonb commented on playbooks/roles/ecommerce/defaults/main.yml:
```diff
@@ -69,6 +69,10 @@ ECOMMERCE_PAYPAL_CLIENT_ID: 'SET-ME-PLEASE'
 ECOMMERCE_PAYPAL_CANCEL_URL: '{{ ECOMMERCE_LMS_URL_ROOT }}/commerce/checkout/cancel/'
+ECOMMERCE_PAYPAL_WEB_PROFILE_CONFIG:
+    openedx_default:
+      brand_name: 'OpenEdX'
```
We should choose a default value that does not involve edX to avoid potential confusion.

jimabramson replied:
do you mean for the brand name, or for the profile key?

clintonb replied:
Brand name.

jimabramson commented:
@maxrothman please review

jimabramson commented:
@clintonb fixed brand_name (to match the platform name default)

clintonb commented:
:+1:

rlucioni commented:
Your changes in [#177](https://github.com/edx/ecommerce/pull/177) include a "presentation" key in the dictionary. Why is that missing here?

jimabramson commented:
@rlucioni nice catch, i inadvertently omitted it here.

rlucioni commented on playbooks/roles/ecommerce/defaults/main.yml:
```diff
@@ -69,6 +69,11 @@ ECOMMERCE_PAYPAL_CLIENT_ID: 'SET-ME-PLEASE'
 ECOMMERCE_PAYPAL_RECEIPT_URL: '{{ ECOMMERCE_LMS_URL_ROOT }}/commerce/checkout/receipt/'
 ECOMMERCE_PAYPAL_CANCEL_URL: '{{ ECOMMERCE_LMS_URL_ROOT }}/commerce/checkout/cancel/'
+ECOMMERCE_PAYPAL_WEB_PROFILE_CONFIG:
+    openedx_default:
```
In production use, we won't be using an "openedx_default" web profile; we'll have substituted our own brand name and logo image. This is admittedly a minor point, but would you be in favor of changing this key to simply read "default"? It's implied that the defaults provided here are intended for users of Open edX.

jimabramson replied:
that makes sense, i will change this in both PRs.

rlucioni commented:
Aside from my proposed simplification of the key name, :+1: .

maxrothman commented:
:+1: What should those values be in prod/stage/edge/etc.?

rlucioni commented:
@jimabramson please record your answer to @maxrothman's question on the [Service Settings wiki page](https://openedx.atlassian.net/wiki/display/XCOM/Service+Settings).

jimabramson commented:
squashed.   will merge after build.

@maxrothman I updated https://openedx.atlassian.net/wiki/display/XCOM/Service+Settings with the values for prod and stage.  I don't believe we need to support edge at this time.  Note the "logo image" values point to absolute urls at our marketing sites.  Is this OK?

jimabramson commented:
@maxrothman  I've made changes pursuant to our conversation earlier.  I've also updated with settings wiki linked above.  As of this change, out of the box and with configuration defaults, we have a generic uncustomized openedx branding profile for PayPal checkout pages; in staging and production the image and brand name are overridden for edx.org.  The LMS' hosted assets are used as the image source.

@rlucioni @clintonb FYI.

jimabramson commented:
abandoning this PR as we won't use configuration to manage this.
